terraform/main_vm_start.sh

```diff
@@ -0,0 +1,99 @@
+#!/bin/bash
+
+TIME_LIMIT=1800
+
+# apt_install command_to_check package_name
+function apt_install {
+    while [[ ! `command -v $1` ]];
+    do
+        apt-get -y update
+        apt-get -y install $2
+        sleep 3
+    done
+}
+
+mkdir go
+export HOME=/
+cd $HOME/go
+echo "`date -u` - script start time" > $HOME/results.txt
+SECONDS=0
+
+apt_install pip3 python3-pip
+apt_install mysql mysql-client
+
+snap install --classic --channel=1.14/stable go
+
+export GOPATH=$HOME/go
+export PATH=/snap/bin:$GOPATH/bin:$PATH
+
+/snap/bin/go get -v -t github.com/Alexandr-TS/fleetspeak/...
+
+cd /go/src/github.com/Alexandr-TS/fleetspeak/
+git checkout prep_cloud
```

Comment:
After this PR is approved, these lines should be replaced with
`/snap/bin/go get -v -t github.com/google/fleetspeak/...`
`cd /go/src/github.com/google/fleetspeak/`